.github/workflows/lint.yml

```diff
@@ -0,0 +1,17 @@
+name: run black and isort
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install isort and black
+        run: pip install isort black
+      - name: Run black
+        run: black --check --diff .
+      - name: Run isort
+        run: isort --profile black --check --diff .
```

Comment:
Any reason why we are not sticking with the default profile? I'm curious.

Reply:
Yes, the same issue seems to persists if we stick with the default profile I can produce logs for that as well. The only way I got both of them to work in harmony is by setting isort's profile to black.

Reply:
Makes sense.